Source/MQTTnet/Adapter/MqttChannelAdapter.cs

```diff
@@ -33,8 +33,7 @@ public sealed class MqttChannelAdapter : Disposable, IMqttChannelAdapter
 
         readonly AsyncLock _syncRoot = new AsyncLock();
 
-        long _bytesReceived;
-        long _bytesSent;
+        Statistics _statistics;     // mutable struct, don't make readonly!
```

Comment:
Statistics moved to separate struct. Perf-wise there's no difference, only for hygiene.
And if there might be some better way to reset* in the future, it's only one place to update.

\* e.g. if the types were `int` instead of `long` one could use a "union" of the two ints overlayed with a long, then the reset would just be a volatile write to that long with 0.
Note: I don't propose to change to int, just as example.